src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -1834,4 +1736,82 @@ public void action() throws SaveException {
             }
         }
     }
+
+    private class GenerateKeysAction implements BaseAction {
+
+        List<BibEntry> entries;
+        int numSelected;
+        boolean canceled;
+
+        // Run first, in EDT:
```

Comment:
That comment is obsolete now, right?>
Could this be extracted to a new file?

Reply:
Done. You are right, the comment was a left-over.